### ISSUE TITLE
Get-AzureRmApiManagementSubscription ByProductAndUser

### ISSUE THREAD

**Issue by krispenner** [Azure/azure-powershell#8671]
## Description of the new feature
Allow specifying both -ProductId and -UserId on the Get-AzureRmApiManagementSubscription cmdlet to return a narrowed list of subscriptions.

**Comment by cormacpayne** [comment]
@solankisamir Hey Samir, would you mind taking a look at this feature request?

**Comment by solankisamir** [comment]
ack

**Comment by solankisamir** [comment]
This is possible in the REST API. Will work on adding support in Powershell

```
GET /subscriptions/subid/resourceGroups/rgname/providers/Microsoft.ApiManagement/service/apimServiceName/subscriptions?api-version=2019-01-01&$filter=properties/scope%20eq%20'%2Fproducts%2F551ee9ddbb739f72ec7f0c40'%20and%20properties/ownerId%20eq%20'551ee9debb739f72ec7f0c42' HTTP/1.1
Authorization: Bearer XXXXX
Host: management.azure.com
```

**Comment by solankisamir** [comment]
released - [v2.4.0-July2019](https://github.com/Azure/azure-powershell/releases/tag/v2.4.0-July2019)
